frame/system/src/lib.rs

```diff
@@ -1272,6 +1272,40 @@ impl<T: Config> Pallet<T> {
 	/// Remove temporary "environment" entries in storage, compute the storage root and return the
 	/// resulting header for this block.
 	pub fn finalize() -> T::Header {
+		log::debug!(
+			target: "runtime::system",
+			"[{:?}] length: {} (normal {:?}, op: {:?}, mandatory {:?}) / normal weight: {} ({:?}) \
```

Comment:
```suggestion
			"[{:?}] ext num: {} (normal {:?}, op: {:?}, mandatory {:?}) / normal weight: {} ({:?}) \
```

Reply:
hmm no, this is extrinsic length, not count.

Reply:
:facepalm: